certbot/certbot/_internal/tests/crypto_util_test.py

```diff
@@ -340,7 +340,7 @@ def _call(cls, *args, **kwargs):
         return get_sans_from_cert(*args, **kwargs)
 
     def test_single(self):
-        assert [] == self._call(test_util.load_vector('cert_512.pem'))
+        assert ['example.com'] == self._call(test_util.load_vector('cert_512.pem'))
```

Comment:
🤷 can't figure out what this was doing initially, so this seems fine. this is why we don't do 1800-line PRs.